notary-server/config.json

```diff
@@ -0,0 +1,19 @@
+{
+	"server": {
+		"addr": ":4443"
+	},
+	"trust_service": {
+	  "type": "remote",
+	  "hostname": "notarysigner",
+	  "port": "7899",
+	  "tls_ca_file": "./fixtures/root-ca.crt",			
+	  "key_algorithm": "ecdsa"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "mysql",
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
```

Comment:
I think we should remove this from the config file and and require users to set the NOTARY_SERVER_STORAGE_DB_URL environment variable.

I wonder if this container can me made to work with the official mysql image. There's a notarymysql container that lives in the notary repo right now, but it would be great not to have to create yet another official image. The notarymysql container is just mysql with a database precreated.

I saw that the wordpress image uses environment variables for mysql settings, and a launch script creates the database if it doesn't already exist:

https://github.com/docker-library/wordpress/blob/master/apache/docker-entrypoint.sh

Is this an example to follow?